### DISCUSSION TITLE
What's the difference between rspack and rsbuild? If I want to build a react component library, which one should i choose?

### DISCUSSION THREAD

**Discussion by wood3n** [web-infra-dev/rsbuild#1797] [Q&A] [answered]
As the title

**Comment by chenjiahan** [comment, marked as answer]
See also: https://rsbuild.dev/guide/faq/general#can-rsbuild-be-used-to-build-libraries-or-ui-components

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/6ebd03bc-f94e-4fe1-a4da-73c2dc670db7)

And we are working on a new tool named `Rslib` based on Rsbuild, which will provide the library build mode. see: https://github.com/web-infra-dev/rsbuild/discussions/1678

**Reply by chenjiahan** [reply]
The Rslib repository is now public: https://github.com/web-infra-dev/rslib
